Tests/ServiceRepositoryTest.php

```diff
@@ -14,38 +14,38 @@
 use Fixtures\Bundles\RepositoryServiceBundle\Document\TestDefaultRepoDocument;
 use Fixtures\Bundles\RepositoryServiceBundle\Repository\TestCustomClassRepoRepository;
 use Fixtures\Bundles\RepositoryServiceBundle\Repository\TestCustomServiceRepoRepository;
+use Fixtures\Bundles\RepositoryServiceBundle\Repository\TestUnmappedDocumentRepository;
 use Fixtures\Bundles\RepositoryServiceBundle\RepositoryServiceBundle;
+use LogicException;
 use Symfony\Component\DependencyInjection\Alias;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Definition;
 use Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
 
 class ServiceRepositoryTest extends TestCase
 {
+    /** @var ContainerBuilder */
+    private $container;
+
     protected function setUp()
     {
         parent::setUp();
 
-        if (class_exists(DocumentManager::class)) {
-            return;
+        if (! class_exists(DocumentManager::class)) {
+            $this->markTestSkipped('Doctrine MongoDB ODM is not available.');
```

Comment:
I realize this was here before, but why would this happen?

Reply:
This was copied over from ORM - I'll drop it 👍